build/transpile.js

```diff
@@ -1165,7 +1165,6 @@ class Transpiler {
                     line.replace ('asyncio.get_event_loop().run_until_complete(main())', 'main()')
                         .replace ('asyncio.run(main())', 'main()')
                         .replace ('import ccxt.async_support as ccxt', 'import ccxt')
-                        .replace (/.*token\_bucket.*/g, '')
```

Comment:
this line is no longer needed in transpiler, because in python, there is no mention of `tocken_bucket` anywhere, probably there was something the past, and now this interferes with current transpilation